core/src/test/java/org/infinispan/jmx/CacheMgmtInterceptorMBeanTest.java

```diff
@@ -209,11 +213,27 @@ public void testGetAll() throws Exception {
       assertEquals(0.5f, hitRatio);
    }
 
+   private void eventuallyAssertAttributeValue(String attrName, float expectedValue) {
+      eventuallyEquals(expectedValue, () -> {
+         try {
+            String receivedVal = server.getAttribute(mgmtInterceptor, attrName).toString();
+            return Float.parseFloat(receivedVal);
+         } catch (Exception e) {
+            throw Exceptions.propagate(e);
+         }
+      });
+   }
+
    private void assertAttributeValue(String attrName, float expectedValue) throws Exception {
       String receivedVal = server.getAttribute(mgmtInterceptor, attrName).toString();
       assert Float.parseFloat(receivedVal) == expectedValue : "expecting " + expectedValue + " for " + attrName + ", but received " + receivedVal;
    }
 
+   private void eventuallyAssertEvictions(float expectedValue) {
+      eventuallyAssertAttributeValue("Evictions", expectedValue);
+      assert expectedValue == advanced.getStats().getEvictions();
```

Comment:
When updating code can you replace plain asserts with those from the testing framework ?

Reply:
Sure, I can update these.